orcid/test_member_obo.py

```diff
@@ -68,11 +67,17 @@ def test_012_full_scope_obo(self):
 
     def test_013_full_scope_post_work(self):
         response = self.post_member_obo(OauthOpenId.obo_token, self.version, "work", "ma30_work_member_obo.xml")
-        curl_params = ['-L', '-i', '-k', '-H', 'Authorization: Bearer ' + self.obo_token, '-H', 'Accept: application/xml','-X', 'GET']
+        curl_params = ['-L', '-i', '-k', '-H', 'Authorization: Bearer ' + OauthOpenId.obo_token, '-H', 'Accept: application/xml','-X', 'GET']
         url = "api." + properties.test_server + "/v3.0/%s/work/" % (self.orcid_id)
-        search_pattern = "%s(\d+)" % url
-        putcode = re.search(search_pattern, re.sub('[\s+]', '', response))
+        search_pattern = r"%s(\d+)" % url
+        putcode = re.search(search_pattern, re.sub(r'[\s+]', '', response))
         url = "https://" + url + putcode.group(1)
         read_response = self.orcid_curl(url, curl_params)
         assertion_check = "<common:assertion-origin-name>Member OBO Second Client</common:assertion-origin-name>"
-        self.assertTrue(assertion_check in read_response, "Unexpected result: " + read_response)
\ No newline at end of file
+        self.assertTrue(assertion_check in read_response, "Unexpected result: " + read_response)
+        if properties.type != "actions":
```

Comment:
If the test is run locally, it removes one of the added works (that's impossible to remove without an OBO token) and then revokes the token. This allows the test to be run infinitely on the same record, which is really convenient for local testing